core/jvm/src/internal/SerializedValue.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019-2024 JetBrains s.r.o. and contributors.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.datetime.internal
+
+import kotlinx.datetime.*
+import java.io.*
+
+internal class SerializedValue(var typeTag: Int, var value: Any?) : Externalizable {
```

Comment:
Can these values be `private`?

Reply:
They can, but does it makes a difference for an internal class? Or you mean in order not to generate getters/setters?

Reply:
Yep, for the latter.